portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/snippets/loginImage-snippet.vm

```diff
@@ -28,7 +28,7 @@
         #else
 
             <p class="Mrphs-loginUser__menuitem Mrphs-loginUser__menuitemk--alt">
-                <a href="${loginLogInOutUrl2}" title="${loginMessage2}" id="loginLink2" class="Mrphs-loginUser__message">
+                <a class="button" href="${loginLogInOutUrl2}" title="${loginMessage2}" id="loginLink2" class="Mrphs-loginUser__message">
```

Comment:
Where's the button class coming from?

Reply:
I think it's coming from Morpheus, Bootstrap has .btn .btn-primary etc